internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,65 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	times := strings.Split(data, ",")
+	if len(times) != 2 {
+		return 0, 0, errors.New("Insufficient data")
+	}
+	stepsStr := strings.TrimSpace(times[0])
+	if stepsStr == "" {
+		return 0, 0, errors.New("steps value is empty")
+	}
+
+	steps, err := strconv.Atoi(stepsStr)
+	if err != nil {
+		return 0, 0, errors.New("Error converting the number of steps")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```